Dockerfile.QA

```diff
@@ -70,6 +69,13 @@ RUN wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/nul
     apt-get install -y --no-install-recommends \
       cmake-data=3.21.1-0kitware1ubuntu20.04.1 cmake=3.21.1-0kitware1ubuntu20.04.1
 
+# Install boost version >= 1.78 for boost::span
+# Current libboost-dev apt packages are < 1.78, so install from tar.gz
+RUN wget -O ${TRITONTMP_DIR}/boost.tar.gz \
+        https://boostorg.jfrog.io/artifactory/main/release/1.80.0/source/boost_1_80_0.tar.gz && \
+    (cd ${TRITONTMP_DIR} && tar xzf boost.tar.gz) && \
+    mv ${TRITONTMP_DIR}/boost_1_80_0/boost /usr/include/boost
+
```

Comment:
TODO: check if this is required in QA container. May only be needed as a build time dependency in `build.py`